tokens/lift_testnet.json

```
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "0x189ad84364395faf58df275ee66f417d73571bec",
+    "aurora_address": "0x189ad84364395faf58df275ee66f417d73571bec",
```

The corresponding bridge token address is the following:

```suggestion
    "aurora_address": "0x230922bb4b044a146e8ac662ec348f1889d45881",
```